homeassistant/components/wallbox/helpers.py

```diff
@@ -0,0 +1,59 @@
+"""Helpers for Wallbox integration."""
+
+from datetime import datetime as datetime_sys
+
+from dateutil import tz
+
+import homeassistant.util.dt as dt_util
+
+
+def map_schedule_item_to_ha(item: dict) -> dict:
+    """Map local schedule item to be more usable with HA."""
+    converted = {}
+    converted["start"] = _convert_to_local_time(item["start"])
```

Comment:
what is this by default? always UTC? or depends on wallbox config? hesitant about whether this should be something we do in HomeAssistant (or more upstream (or not))?

Reply:
Wallbox API returns always UTC time.
I thought this quite a while as well, but I decided to covert the time to local for easier use in HA end.

If you see it's better the other way around, it can be changed of course

Reply:
hmmmmmmm, let me think on finding a cleaner way I would like to skip the extra functions